src/MicrosoftTeams.ts

```diff
@@ -54,14 +54,15 @@ namespace microsoftTeams
     }
 
     export interface TabInformation {
-        tabList?: TabInstance[];
+        teamTabs?: TabInstance[];
     }
 
     export interface TabInstance {
         tabName: string;
         entityId?: string;
         channelId?: string;
         channelName?: string;
+        channelIsFavorite?: boolean;
```

Comment:
Do we need a teamIsFavorite property as well or did we decide to not enumerate non-favorited teams at all?

Reply:
great point. let me just add it to future proof this.